### PR TITLE
trivia games joinable midgame + show who submits qs

### PR DESCRIPTION
**Pull request by Redew**
Made it so you can join trivia games midgame. I also made it so they can't join if unable to talk. The last change was showing who submitted the question in QW.

### CODE REVIEW

File: chat-plugins/trivia.js [hunk 1]
```diff
@@ -198,7 +198,7 @@ var Trivia = (function () {
 	};
 
 	Trivia.prototype.startGame = function (output) {
-		if (this.phase !== 'signup') return output.sendReply("There is no trivia game in its signup phase.");
+		if (this.phase !== 'signup') return output.sendReply("There is no trivia game to join.");
```
Comment (ascriptmaster) [inline chat-plugins/trivia.js]: Since this is ostensibly the function to start the game, why does the message say there is no trivia game to *join*?

Comment (Redew) [inline chat-plugins/trivia.js]: Oops, you're right. That's a goof on my part. I will edit it to say start instead of join.

**Comment by Redew** [comment]
@ascriptmaster I squashed this if you want to check.

File: chat-plugins/trivia.js [hunk 2]
```diff
@@ -164,7 +164,7 @@ var Trivia = (function () {
 	}
 
 	Trivia.prototype.addParticipant = function (output, user) {
-		if (this.phase !== 'signup') return output.sendReply("There is no trivia game in its signup phase.");
+		if ((user.locked || this.room.isMuted(user)) && !user.can('bypassall')) return this.sendReply("You cannot do this while unable to talk.");
```
Comment (ascriptmaster) [inline chat-plugins/trivia.js]: Hmmm, I think we had a function called `this.canTalk` that does all the checks of whether a user can talk in a room. You might want to look into that.

**Comment by Redew** [comment]
@ascriptmaster adding ```canTalk``` to ```addParticipant``` wasn't the best idea, so I moved it to the ```join``` command.

**Pull request merged**